src/main/java/org/example/bookstore/model/User.java

```diff
@@ -0,0 +1,31 @@
+package org.example.bookstore.model;
+
+import jakarta.persistence.Column;
+import jakarta.persistence.Entity;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.Id;
+import jakarta.persistence.Table;
+import lombok.Getter;
+import lombok.Setter;
+import lombok.ToString;
+
+@Entity
+@Getter
+@Setter
+@ToString
+@Table(name = "users")
+public class User {
```

Comment:
I believe soft delete should be applied to user as well

Reply:
There is no soft delete requirement for the user entity, and in general, we do not usually delete users, and we won't have a delete endpoint for it in this project. But I think you can still leave it as it is if you want